README.md

```diff
@@ -16,7 +16,7 @@ This is the screenshot of my page
 - Flexbox and CSS Grid Layout
 ## Live Demo
 
-[Live Demo Link](https://have-samuel.github.io/portfolio1/)
+[Live Demo Link](https://git@github.com:Have-Samuel/portfolio1.git)
```

Comment:
- [ ] Hello @Have-Samuel, I have noticed that the link added here is not the one needed for GitHub pages deployment. Please replace it with the GitHub pages link for this portfolio project.